pkg/ansible/watches/watches.go

```diff
@@ -321,15 +307,28 @@ func Load(path string, maxWorkers, ansibleVerbosity int) ([]Watch, error) {
 		return nil, err
 	}
 
-	watches := []Watch{}
-	err = yaml.Unmarshal(b, &watches)
+	// First we unmarshall to the alias structure
+	alias := []alias{}
+	err = yaml.Unmarshal(b, &alias)
 	if err != nil {
 		log.Error(err, "Failed to unmarshal config")
 		return nil, err
 	}
 
+	// We copy contents from alias structure to the watch structure
+	watches := []Watch{}
+	for _, a := range alias {
+		w, _ := buildWatch(a)
+		watches = append(watches, w)
+	}
+
 	watchesMap := make(map[schema.GroupVersionKind]bool)
 	for _, watch := range watches {
+
+		if err != nil {
+			log.Error(err, "Failed to get config file")
+			return nil, err
+		}
```

Comment:
This doesn't make any sense here. Where is `err` coming from? I want to say this code is misplaced.

Reply:
Was trying something and forgot to remove that piece